README.md

```diff
@@ -316,3 +316,14 @@ The integration tests live in the `integration` project. Consult the integration
 
 In the early days of the project, there were JUnit-based integration tests. We are in
 process of migrating them to Test Runner.
+
+#### Cleaning up workspaces in tests
+
+For integration tests, don't need to delete workspaces because it's taken care of
```

Comment:
TestRunner isn't integrated with Janitor so it isn't doing this cleanup (though it could if we were testing against live environments and we built that integration). The cleanup is currently part of the tests (though since it's included in the `WorkspaceAllocateTestScriptBase` base class, developers don't need to think about it while writing tests)